source/include/core_sntp_client.h

```diff
@@ -157,6 +157,11 @@ typedef struct NetworkContext NetworkContext_t;
  * @brief Interface for user-defined function to send data to the network
```

Comment:
Might be useful to say that this function will only send a single UDP datagram.

Reply:
Sure